src/main/java/com/uipath/uipathpackage/UiPathTest.java

```diff
@@ -301,8 +306,22 @@ public TraceLevel getTraceLevel() {
     public String getParametersFilePath() {
         return parametersFilePath;
     }
+    
+    /**
+     * attachRobotLogs
+     *
+     * @return boolean attachRobotLogs
+     */
+    public boolean getAttachRobotLogs() {
+		return attachRobotLogs;
+	}
 
-    private void validateParameters() throws AbortException {
+    @DataBoundSetter
+    public void setAttachRobotLogs(boolean attachRobotLogs) {
```

Comment:
Why does the getter above have a documentation header and why do we not have the documentation header for this setter?

Reply:
done